tests/test_profiler.py

```diff
@@ -407,6 +494,9 @@ def on_fit_end(self, trainer, *args, **kwargs) -> None:
     assert profiler._output_file is None
 
 
+@pytest.mark.skipif(_TORCH_GREATER_EQUAL_1_8, reason="currently not supported for PyTorch 1.8")
 def test_pytorch_profiler_deepcopy(pytorch_profiler):
+    pytorch_profiler.start("on_train_start")
+    torch.tensor(1)
```

Comment:
Why does it not work for 1.8?

And remove the start and extra tensor?

Reply:
`Why does it not work for 1.8?`. It is using the legacy profiler and shouldn't
remove the start: It is needed so the profiler gets created
extra tensor: It is needed for at least 1 events to be recorded. Can be removed with extra checks.